libcontainer/container_linux.go

```diff
@@ -21,6 +21,7 @@ import (
 	"golang.org/x/sys/unix"
 
 	"github.com/opencontainers/runc/libcontainer/cgroups"
+	cgConfig "github.com/opencontainers/runc/libcontainer/cgroups/configs"
```

Comment:
do you know if these types are also imported elsewhere _separate from_ the `cgroups` package itself?  considering if that was  not the case, these types could potentially  be merged with the cgroups package 🤔

Reply:
Good question.

Looks like there's no need to have a separate package for cgroups-related structures.

The PR is now updated to use libcontainer/cgroups. The only negative effect is the patch is now bigger (had to replace `configs` with `cgroups` everywhere.

PTAL @thaJeztah 🙏🏻

Reply:
Thanks! I wanted to give the changes a quick try to see if nothing bad fell out of it, so I opened some  test-PRs in moby to double-check;

- https://github.com/moby/moby/pull/49095